kitsune/search/v2/documents.py

```diff
@@ -170,14 +186,38 @@ def prepare_is_solution(self, instance):
     def prepare_locale(self, instance):
         return instance.question.locale
 
+    def prepare_num_helpful_votes(self, instance):
+        if hasattr(instance, "es_num_helpful_votes"):
+            return instance.es_num_helpful_votes
+        return instance.num_helpful_votes
+
+    def prepare_num_unhelpful_votes(self, instance):
+        if hasattr(instance, "es_num_unhelpful_votes"):
+            return instance.es_num_unhelpful_votes
+        return instance.num_unhelpful_votes
+
     def get_field_value(self, field, instance, *args):
         if field.startswith("question_"):
             instance = instance.question
         return super().get_field_value(field, instance, *args)
 
     @classmethod
     def get_model(cls):
-        return question_models.Answer
+        return Answer
+
+    @classmethod
+    def get_queryset(cls):
+        return (
+            Answer.objects
+            # prefetch each answer's question,
+            # applying the same optimizations as in the QuestionDocument
+            .prefetch_related(Prefetch("question", queryset=super().get_queryset()))
```

Comment:
Since we are using `classmethod` I am wondering if it's better/cleaner to call here `QuestionDocument.get_queryset()` instead of super